### ISSUE TITLE
After I close excel, NVDA fails to announce items on desktop and other areas

### ISSUE THREAD

**Issue by the-swissionary** [nvaccess/nvda#10137]
### Steps to reproduce:
1) Run NVDA.
2) Run a spreadsheet in MIcrosoft Excel. Wait until Excel is open and NVDA displays the contents of the file.
3) Close excel (I usually use ALT+F4)
4) Press WINDOWSKEY+D to access the Desktop.
5) Using the arrow key, try navigating amongst the icons on the desktop.
### Actual behavior:
As soon as Excel is closed, NVDA won't announce the names of the icons on my desktop anymore. Many other things, such as Pop-Up Menus or Files within Explorer List Views are no longer announced either. To get it running again, I usually have to restart NVDA.
Please note that Excel is not the only app causing this. It frequently happens when I try to use MS Edge. Excel is just an example on which I can reliably reproduce the issue.

### Expected behavior:
I expect NVDA to announce the different icons in the usual way.

#### NVDA installed/portable/running from source:
Installed latest published version
#### NVDA version:
2019.2
#### Windows version:
Windows Version	10.0.18362 Build 18362

#### Name and version of other software in use when reproducing the issue:
Microsoft® Excel® für Office 365 MSO (16.0.11328.20362) 32-Bit 
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have only recently started using MS Excel. The issue was already present in NVDA 2019.1 and has not changed with the latest upgrade.

**Comment by the-swissionary** [comment]
[nvda-excelbug.log](https://github.com/nvaccess/nvda/files/3525926/nvda-excelbug.log)
Added a Log File which shows the log while I reproduce the above steps. I close Excel shortly after 16:43 I think

**Comment by lukaszgo1** [comment]
Could you disable Audio Themes add-on, and then try to reproduce the issue?

**Comment by Brian1Gaff** [comment]
Log out of your  current account in windows, then log back in again.

I've had this problem before its as if something screws around with the 
windows info being sent to screenreaders. I'm surprised it works if you 
restart nvda though, normally it does not or is very hit and miss. What 
version of windows do you use and what version of  the spreadsheet and have 
you found sometimes something like Adobe reader causes similar issues?
 What happens if you  start a portable copy of nvda  up,
Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by Brian1Gaff** [comment]
I notice that on windows 7 with a slightly modified version of that add on I 
see no such errors. It could well be that windows 7 is more picky, but not 
sure.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by the-swissionary** [comment]
> Could you disable Audio Themes add-on, and then try to reproduce the issue?

Dear Me.... That indeed solved the issue for me. Or in other words: With that plugin disabled, I can't reproduce the buggy behaviour and NVDA does what it's supposed to do.

**Comment by the-swissionary** [comment]
> 
> What version of windows do you use and what version of the spreadsheet and have you found sometimes something like Adobe reader causes similar issues?
I'm Using Windows10 pro 64Bit (latest) and MS Excel for Office 365 16.0.1
No, Adobe Reader hardly ever causes this behaviour to happen on my system.
> What happens if you start a portable copy of nvda up?
I'll have to try this once I've got more time.
As for now, I guess I'll just keep that Audio Themes Plug In disabled (possibly report the bug on their github), which seems to solve the issue for me.

**Comment by lukaszgo1** [comment]
There is nothing like 'their GitHub' in regard to the Audio Themes add-on - i.e. it is abandoned. @josephsl given how much troubles this add-on causes would it be reasonable to either found someone to maintain it - it seems to be quite popular though I cannot understand this or if no one is willing to do maintenance remove it from the add-on repository for the time being. In my view it is  very important not to have such plugins in the official repository of add-ons it sends very bad message. BTW it might be worth considering to expand issue template to ask user if he/she tried with add-ons disabled.

**Comment by lukaszgo1** [comment]
I should have checked before writing. Audio Themes has a GitHub repository, however it isn't maintained.

**Comment by Adriani90** [comment]
Since this issue is caused by an addon, closing as works for me.

**Comment by XLTechie** [comment]
That add-on is known to be a bit problematic with more recent NVDA versions. 
Actually, it always had some issues--for example, it made NVDA significantly 
more sluggish under certain circumstances.

I absolutely agree with the suggestion of adding the "Does the problem persist 
if you restart NVDA with add-ons disabled?"
